etc/config/c++.defaults.properties

```diff
@@ -12,8 +13,24 @@ binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_sta
 stubRe=\bmain\b
 stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
 
+# AP: hard-code this to my MSVC install
+################################
+# Windows Compilers
+group.cl.compilers=&cl19
+group.cl.needsMulti=false
+group.cl.compilerType=WSL-CL
+group.cl.versionFlag=/?
+group.cl.versionRe=^Microsoft \(R\) C/C\+\+.*$
+#group.cl19.compilers=cl19_64:cl19_32:cl19_arm
+#group.cl19.options=/I/opt/compiler-explorer/windows/10.0.10240.0/ucrt/ /I/opt/compiler-explorer/windows/19.10.25017/lib/native/include/
+#compiler.cl19_64.exe=/opt/compiler-explorer/windows/19.10.25017/lib/native/bin/amd64/cl.exe
+#compiler.cl19_64.name=x86-64 CL 19 2017 RTW
+#compiler.cl19_32.exe=/opt/compiler-explorer/windows/19.10.25017/lib/native/bin/amd64_x86/cl.exe
+#compiler.cl19_32.name=x86 CL 19 2017 RTW
+#compiler.cl19_arm.exe=/opt/compiler-explorer/windows/19.10.25017/lib/native/bin/amd64_arm/cl.exe
+#compiler.cl19_arm.name=ARM CL 2017 RTW
 
-#################################
-#################################
-# Installed libs (See c++.amazon.properties for a scheme of libs group)
-libs=
```

Comment:
This deleted lines should probably stay. Local users might use this as a template for their .local and a reminder that you can add your own libs is never bad

Reply:
That was a mistake, thank you for noticing!